include/seqan3/utility/char_operations/pretty_print.hpp

```diff
@@ -0,0 +1,70 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \brief Provides pretty printing for chars.
```

Comment:
I know that this is not the scope of this PR, however users may not know what we consider as "pretty printing". Can we find a higher level description, e.g. "Provides the transformation of chars to a visual representation/ to a printable string" or similar?

Reply:
How about:

```suggestion
 * \brief Provides seqan3::detail::make_printable which converts non printable characters (e.g. a tab) to a std::string.
```

Reply:
Sounds great!